hyperjumptech/monika
# Feat: 362 add notification info

# Monika Pull Request (PR)  
Fix #362 Get More Info for Monika

## What feature/issue does this PR add  
To get more info where monika is running

## How did you implement / how did you fix it  
1.  add city and isp info into alert and notification message
2. fix send notification when monika is terminated

## How to test  
1. set monika configurations with teams, mailgun and desktop notif
2. run monika with notifications config
3. it should show city and isp information also:
```
Monika is running on Bandung - PT. TELKOM INDONESIA (36.79.170.193) - muslims-MacBook-Air.local (192.168.1.5)
```

## result image
![Screen Shot 2021-09-07 at 23 01 06](https://user-images.githubusercontent.com/8952493/132376137-d37bd959-6534-49c3-a49f-d2797c830364.png)
